source/server/listener_manager_impl.h

```diff
@@ -106,14 +115,17 @@ class ListenerManagerImpl : public ListenerManager, Logger::Loggable<Logger::Id:
 
 // TODO(mattklein123): Listener manager stats.
 // TODO(mattklein123): Check that addresses for unbound listeners are unique.
-// TODO(mattklein123): Real listener draining with a per listener drain manager.
 // TODO(mattklein123): Detect runtime worker listener addition failure and handle.
+// TODO(mattklein123): Consider getting rid of pre-worker start and post-worker start code by
+// initializing all listeners after workers are started. This is related to correctly dealing with
```

Comment:
I prefer lining up but I think when all the TODOs got reformatted a while ago the lining up got removed. I will reline up.